gson/src/test/java/com/google/gson/ExposeAnnotationExclusionStrategyTest.java

```diff
@@ -28,6 +28,14 @@
  *
  * @author Joel Leitch
  */
+class ClassWithNoFields {
+  // Nothing here..
+  @Override
+  public boolean equals(Object other) {
+    return other.getClass() == TestTypes.ClassWithNoFields.class;
+  }
+}
+
```

Comment:
guess this was left by mistake here

Reply:
I accidentally removed it then added it back :s

Reply:
fixed.